www-apps/gogs/gogs-0.9.141.ebuild

```diff
@@ -105,7 +104,7 @@ KEYWORDS="~amd64 ~x86"
 
 IUSE_CACHE_ADAPTER="redis memcached"
 IUSE_DATABASE_ADAPTER="mysql postgres +sqlite tidb"
-IUSE="$IUSE_DATABASE_ADAPTER $IUSE_CACHE_ADAPTER pam +ssl"
+IUSE="$IUSE_DATABASE_ADAPTER $IUSE_CACHE_ADAPTER pam +ssl systemd"
```

Comment:
The USE `systemd` is not allowed any more to conditionally install rc files. It's forbidden by new EAPIs.